umap/static/umap/js/modules/utils.js

```diff
@@ -416,9 +416,11 @@ export function loadTemplate(html) {
 }
 
 export function loadTemplateWithRefs(html) {
-  const element = loadTemplate(html)
+  const template = document.createElement('template')
+  template.innerHTML = html
+  const element = template.content.firstElementChild
   const elements = {}
-  for (const node of element.querySelectorAll('[data-ref]')) {
+  for (const node of template.content.querySelectorAll('[data-ref]')) {
```

Comment:
Or maybe `loadTemplate` should (also) return the `template.content`?

Reply:
IMHO, that would pollute the API, as we would have to do this each time:

```
const myElement = Utils.loadTemplate('xxx').firstElementChild
```

instead of what we do now:

```
const myElement = Utils.loadTemplate('xxx')
```